modules/creating-infra-machines.adoc

```diff
@@ -74,9 +94,18 @@ rendered-worker-4f110718fe88e5f349987854a1147755            365c1cfd14de5b0e3b85
 rendered-worker-afc758e194d6188677eb837842d3b379            02c07496ba0417b3e12b78fb32baf6293d314f79   2.2.0             31d
 rendered-worker-daa08cc1e8f5fcdeba24de60cd955cc3            365c1cfd14de5b0e3b85e0fc815b0060f36ab955   2.2.0             13d
 ----
++
+You should see a new machine config, with the `rendered-infra-*` prefix.
 
 . Optional: To deploy changes to a custom pool, create a machine config that uses the custom pool name as the label, such as `infra`. Note that this is not required and only shown for instructional purposes. In this manner, you can apply any custom configurations specific to only your infra nodes.
 +
+[NOTE]
+====
+After you create a machine config, the MCO reboots the nodes that are associated with the machine config to apply the configuration.
```

Comment:
`After you create a machine config, the MCO reboots the nodes that are associated with the machine config to apply the configuration.` -> `After the new MCP is created, the MCO generates a new rendered config for that pool, and associated nodes of that pool will reboot to apply the new configuration`